lib/gui/menu.js

```diff
@@ -18,99 +18,139 @@
 
 const electron = require('electron')
 
-const menuTemplate = [
-  {
-    role: 'editMenu'
-  },
-  {
-    label: 'View',
-    submenu: [
-      {
-        role: 'toggledevtools',
-        click () {
-          const window = electron.BrowserWindow.getFocusedWindow()
-          if (window) {
-            window.webContents.openDevTools({
-              mode: 'detach'
-            })
+/* eslint-disable no-magic-numbers */
+
+/**
+ * @summary Builds a native application menu for a given window
+ * @param {electron.BrowserWindow} window - BrowserWindow instance
+ * @example
+ * buildWindowMenu(mainWindow)
+ */
+const buildWindowMenu = (window) => {
+  /**
+   * @summary Toggle the main window's devtools
+   * @example
+   * toggleDevTools()
+   */
+  const toggleDevTools = () => {
+    if (!window) {
+      return
+    }
+
+    // NOTE: We can't use `webContents.toggleDevTools()` here,
+    // as we need to force detached mode
```

Comment:
Would it also be possible to force it to open on the "Console" tab? :)

Reply:
Nope, there's no option for that